lxd/cluster/gateway.go

```diff
@@ -742,6 +742,15 @@ func (g *Gateway) LeaderAddress() (string, error) {
 	return "", fmt.Errorf("RAFT cluster is unavailable")
 }
 
+// NetworkUpdateCert sets a new network certificate for the gateway
```

Comment:
I'd probably just got with `lxd/cluster: Add NetworkUpdateCert` for this one to keep it from overflowing :)